Ziffity/AddCustomer/Console/Command/CreateCustomers.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
```

Comment:
@msakila532 add declare(strict_types=1); in all php files

Reply:
Added